src/validation.h

```diff
@@ -311,11 +311,11 @@ void FlushStateToDisk();
 void PruneAndFlush();
 
 /** (try to) add transaction to memory pool **/
-bool AcceptToMemoryPool(CTxMemPool& pool, CValidationState &state, const CTransaction &tx, bool fLimitFree,
+bool AcceptToMemoryPool(CTxMemPool& pool, CValidationState &state, const std::shared_ptr<const CTransaction> &tx, bool fLimitFree,
```

Comment:
Saw the same thing in 3 other places. I can tag them for you, but didn't wanna spam in case I was misunderstanding.